### DISCUSSION TITLE
Unit 3 is live!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#12] [Announcements]
Hey all,

Just wanted to share a quick note that Unit 3 is live now at https://lightning.ai/pages/courses/deep-learning-fundamentals/.

Unit 3 focuses on
- computation graphs
- full-batch vs stochastic gradient descent
- automatic differentiation
- the PyTorch API

Hope you like it.

And Happy Holidays! 😊
